frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -123,7 +124,7 @@ export const NavBar = withRouter<NavBarProps>(({pages, basePath, hideDivider}) =
   basePath = basePath.replace(/\/$/, '');
 
   const primaryTabs = <ul className="co-m-horizontal-nav__menu-primary">{
-    pages.filter(({href}, i, all) => before.includes(href) || before.includes(_.get(all[i + 1], 'href'))).map(({name, href}) => {
+    pages.filter(({href, primary}, i, all) => before.includes(href) || before.includes(_.get(all[i + 1], 'href')) || primary === true).map(({name, href}) => {
```

Comment:
Do we need this `primary` flag? It looks to me like the tabs display the way we want without using `primary`.

Reply:
Unfortunately, they do not.  With the `primary` flag I get primary tabs, denoted by the thick underline:
![image](https://user-images.githubusercontent.com/12733153/60973770-e0e0e380-a2f6-11e9-9ab6-eb709092f1ea.png)

Removing the `primary` flag causes tabs to be rendered as secondary tabs, denoted by the thin underline:
![image](https://user-images.githubusercontent.com/12733153/60973848-feae4880-a2f6-11e9-934a-418121a9eb99.png)

This is because any tab named 'yaml' causes some special processing in [horizontal-nav.tsx](https://github.com/openshift/console/blob/master/frontend/public/components/utils/horizontal-nav.tsx#L121) which I didn't want to mess with since it is used through out the UI.  Thought `primary` flag was cleanest/easiest to implement.

Reply:
Ah OK.

When we had tabs on that page before, we didn't use `NavBar` (becuase `NavBar` enforces a UX pattern, but that doesn't actually match what we want for the monitoring page). I think it will be less confusing to do that again (see https://github.com/openshift/console/pull/740/files).

Also, we can probably reuse [`SimpleTab`](https://github.com/openshift/console/blob/master/frontend/public/components/utils/simple-tab-nav.tsx#L5).

Reply:
Hi @kyoto, I updated based on https://github.com/openshift/console/pull/740/files, worked great.  I still needed to add routes to `app-contents.tsx` for the three tab pages in order for the match URL to work, else it always was just `/monitoring`.